plugins/dominant-color-images/helper.php

```diff
@@ -49,10 +49,31 @@ function dominant_color_set_image_editors( array $editors ): array {
  * @return array{ has_transparency?: bool, dominant_color?: string }|WP_Error Array with the dominant color and has transparency values or WP_Error on error.
  */
 function dominant_color_get_dominant_color_data( int $attachment_id ) {
-	$mime_type = get_post_mime_type( $attachment_id );
-	if ( 'application/pdf' === $mime_type ) {
-		return new WP_Error( 'no_image_found', __( 'Unable to load image.', 'dominant-color-images' ) );
+	$mime_type            = get_post_mime_type( $attachment_id );
+	$supported_mime_types = array(
+		'image/jpeg',
+		'image/png',
+		'image/gif',
+		'image/webp',
+		'image/avif',
+	);
+
+	/**
+	 * Filter supported mime types for dominant color extraction.
+	 *
+	 * Filter the array of supported mime types for dominant color extraction, by default the plugin
+	 * supports image types supported by WordPress Core.
+	 *
+	 * @since x.x.x
```

Comment:
These `n.e.x.t` versions automatically get replaced prior to a release.

```suggestion
	 * @since n.e.x.t
```